### PR TITLE
Add created/opted in apps to `goal account list`

### PR DESCRIPTION
**Pull request by algomaxj**
## Summary

It's not very pretty, but it matches what we do currently for assets. Here is example output:

```
[online]	VMQAA7D43Y6SZ7SJNMA6EPFEQ3DGD575OEKV4FP5EVZWSCRHKVLXZXQFZE	VMQAA7D43Y6SZ7SJNMA6EPFEQ3DGD575OEKV4FP5EVZWSCRHKVLXZXQFZE	4999999999995000 microAlgos	[created asset IDs: 4 (1234 foo), 5 (1234 bar)]	[created app IDs: 1, 2]	[opted in app IDs: 1]
```

## Testing

I manually tested this change using a private network.

### CODE REVIEW

**Comment by algorotem** [comment]
I'm not sure I know how to parse it intuitively. 
what is the parenthesis around the assets? and is it the app index? If so, I'd separate it by commas.

File: cmd/goal/accountsList.go [hunk 1]
```diff
@@ -231,6 +231,21 @@ func (accountList *AccountsList) outputAccount(addr string, acctInfo v1.Account,
 		}
 		fmt.Printf("]")
 	}
+	if len(acctInfo.AppParams) > 0 {
+		fmt.Printf("\t[created apps:")
+		for aid := range acctInfo.AppParams {
+			fmt.Printf(" %d", aid)
```
Comment (algorotem) [inline cmd/goal/accountsList.go:237]: Oh ok, I see. 
Yes, I'd sperate it with commas.

File: cmd/goal/accountsList.go [hunk 2]
```diff
@@ -231,6 +231,21 @@ func (accountList *AccountsList) outputAccount(addr string, acctInfo v1.Account,
 		}
 		fmt.Printf("]")
 	}
+	if len(acctInfo.AppParams) > 0 {
+		fmt.Printf("\t[created apps:")
```
Comment (algorotem) [inline cmd/goal/accountsList.go:235]: Created apps IDs

File: cmd/goal/accountsList.go [hunk 3]
```diff
@@ -231,6 +231,21 @@ func (accountList *AccountsList) outputAccount(addr string, acctInfo v1.Account,
 		}
 		fmt.Printf("]")
 	}
```
Comment (algorotem) [inline cmd/goal/accountsList.go:233]: I know it's not part of the PR but maybe worth fixing it now. Just add the word ID -> `Created assets IDs`.

File: cmd/goal/accountsList.go (hunk 1 above)
Comment (algomaxj) [reply cmd/goal/accountsList.go:237]: I'm copying exactly what we do for assets. Do we really want commas for apps but spaces for assets?

File: cmd/goal/accountsList.go (hunk 2 above)
Comment (algomaxj) [reply cmd/goal/accountsList.go:235]: done

File: cmd/goal/accountsList.go (hunk 1 above)
Comment (algorotem) [reply cmd/goal/accountsList.go:237]: I think we want commas for both. 
Maybe a good opportunity to fix it.

Comment (algomaxj) [reply cmd/goal/accountsList.go]: alright, did commas and ID for both

File: cmd/goal/accountsList.go (hunk 3 above)
Comment (algomaxj) [reply cmd/goal/accountsList.go:235]: done

File: cmd/goal/accountsList.go [hunk 4]
```diff
@@ -225,12 +226,30 @@ func (accountList *AccountsList) outputAccount(addr string, acctInfo v1.Account,
...
+		var out []string
 		for curid, params := range acctInfo.AssetParams {
-			fmt.Printf(" %d (%d %s)", curid, params.Total, params.UnitName)
+			out = append(out, fmt.Sprintf("%d (%d %s)", curid, params.Total, params.UnitName))
 		}
-		fmt.Printf("]")
+		fmt.Printf("%s]", strings.Join(out, ", "))
 	}
+	if len(acctInfo.AppParams) > 0 {
+		fmt.Printf("\t[created app IDs: ")
```
Comment (tsachiherman) [inline cmd/goal/accountsList.go:237]: nit: unify the 
```
fmt.Printf("\t[created app IDs: ")
...
fmt.Printf("%s]", strings.Join(out, ", "))
```

into 
```
fmt.Printf("\t[created app IDs: %s]", strings.Join(out, ", "))
```

Comment (algomaxj) [reply cmd/goal/accountsList.go]: opted-in assets are not printed here -- they are printed elsewhere along with frozen status, creator, etc.

Comment (algomaxj) [reply cmd/goal/accountsList.go]: done!

**Pull request merged**